deploy/osd-delete-backplane-script-resources/20-delete-backplane-script-resources.CronJob.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: osd-delete-backplane-script-resources
+  namespace: openshift-backplane-managed-scripts
+spec:
+  failedJobsHistoryLimit: 5
+  successfulJobsHistoryLimit: 3
+  concurrencyPolicy: Replace
+  schedule: "* * */1 * *"
```

Comment:
I think this means it runs every minute?

Reply:
indeed, should make it `0 0 */1 * *`.   Or better pick a random time of the day.